euphonic/io.py

```diff
@@ -76,13 +78,17 @@ def _obj_to_dict(obj: T, attrs: Sequence[str]) -> Dict[str, Any]:
     return dout
 
 
-def _process_dict(dictionary: Dict[str, Any], quantities: Sequence[str] = [],
-                  optional: Sequence[str] = []) -> Dict[str, Any]:
+def _process_dict(dictionary: Dict[str, Any],
+                  quantities: Sequence[str]|None = None,
+                  optional: Sequence[str]|None = None) -> Dict[str, Any]:
```

Comment:
Given they're only required to be sequence, why not:
```suggestion
                  quantities: Sequence[str] = (),
                  optional: Sequence[str] = ()) -> Dict[str, Any]:
```

Reply:
Hmm, that _should_ be ok. Let's try it.